src/common/pubsub.ts

```diff
@@ -505,6 +509,26 @@ export const notifyReputationIncrease = async (
     userAfter,
   });
 
+export const notifyCommentDownvoted = async (
```

Comment:
I'd suggest to use the type safe version we have now for workers and messages. not a blocker but ideally as we move forward new implementations should utilize that

Reply:
Yep, I'll move, also 👍 for implementing new workers.

Reply:
be0b580ed2de42e5c010dccd863ca5df4e228d08